ecs-init/volumes/ecs_volume_driver.go

```diff
@@ -15,11 +15,14 @@ package volumes
 
 import (
 	"fmt"
+	"strings"
 	"sync"
 
 	"github.com/cihub/seelog"
 )
 
+const notMountedErrMsg = "not mounted"
```

Comment:
it might be a good idea to use a tool like mountpoint in the future instead of matching on "not mounted". Maybe add a todo?

Reply:
yeah, added.